examples/adder/mandos/adder.scen.json

```diff
@@ -11,7 +11,14 @@
                     "storage": {},
                     "code": ""
                 }
-            }
+            },
+            "newAddresses": [
+                {
+                    "creatorAddress": "0xacc1000000000000000000000000000000000000000000000000000000001234",
+                    "creatorNonce": "5",
+                    "newAddress": "``adder_address"
```

Comment:
use "address:adder_address" or "address:adder_sc". Addresses need to be 32bytes long, this syntax will make sure the right length is generated